app/src/main/java/org/p2p/wallet/common/mvp/BaseFragment.kt

```diff
@@ -77,7 +77,7 @@ abstract class BaseFragment(@LayoutRes layoutRes: Int) : Fragment(layoutRes), Ba
         val window = requireActivity().window ?: return
         with(window) {
             addFlags(WindowManager.LayoutParams.FLAG_DRAWS_SYSTEM_BAR_BACKGROUNDS)
-            statusBarColor = resources.getColor(colorResId, requireActivity().theme)
+            statusBarColor = requireContext().getColor(colorResId)
```

Comment:
Why didn't you inject `ResourcesProvider` and use here?

Reply:
i thought use it only in presenters or interractors

but on fragment side use context itself

Reply:
Got it)

Reply:
It's bad idea to have 2 ways to get color, if you create some provider class, so we need to use it everywhere

Reply:
ok will apply to baseFragment also